packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -274,17 +269,21 @@ export class MonitorATMServicesViewImpl extends React.PureComponent<TPropsWithIn
               <Field
                 onChange={(e, newValue: string) => trackSelectService(newValue)}
                 name="service"
-                component={AdaptedVirtualSelect}
+                component={reduxFormFieldAdapter({ AntInputComponent: Select })}
                 placeholder="Select A Service"
                 props={{
                   className: 'select-a-service-input',
                   value: this.getSelectedService(),
                   disabled: metrics.operationMetricsLoading,
-                  clearable: false,
-                  options: services.map((s: string) => ({ label: s, value: s })),
-                  required: true,
```

Comment:
We no longer require these as

- `clearable` is the default behaviour 
- `options` are being passed as children
- `required` prop is not present 

This applies to all the places

Reply:
what happens to `required` now? Previously the form could not be submitted without required selections.

Reply:
Actually, the implementation of the `Select` element was such that we don't really require the `required` prop

In the `Monitor/ServicesView`, the `Select` component isn't really wrapped inside a form and thus is not being submitted, it's not getting submitted anywhere, we just change the data in charts based on the value of Select

The Select just chooses the first option as the default value, so it's not possible for it to remain empty unless there's no data or service running, in that case, it is not even visible to the user

Reply:
In cases where we are actually submitting the form, for example in the `SearchForm` component, the UI looks something like this when there is no data or services running - 

<img width="1784" alt="Screenshot 2023-08-28 at 4 42 10 AM" src="https://github.com/jaegertracing/jaeger-ui/assets/28570857/6190c6f8-aa5c-45d7-91e2-8a8923b9ebdd">

We just disable the Select as well as the Submit button, and if there were services running, it would just pick the first one as default